README.md

```diff
@@ -145,7 +147,7 @@ end
 
 #### Provider-Specific Records
 
-For provider-specific records (e.g. `ALIAS`), create the record model in `lib/record_store/record` as any other record. In the provider, extend `self.record_types` and append the custom record types to the `Set` returned by `Provider.record_types` (e.g. [`DNSimple.record_types`](https://github.com/Shopify/record_store/blob/1ec0d1410cf8bedf79bc63e8e4cdc7cdb0f1019b/lib/record_store/provider/dnsimple.rb#L5-L7)).
+For provider-specific records (e.g. `ALIAS`), create the record model in `lib/record_store/record` as any other record. In the provider, extend `self.record_types` and append the custom record types to the `Set` returned by `Provider.record_types` (e.g. [`DNSimple.record_types`](https://github.com/Shopify/record_store/blob/e8fb991ae827afb932f97930b55f046aa463d58f/lib/recordstore/provider/dnsimple.rb#L6)).
```

Comment:
@andrewlouis93 https://github.com/Shopify/record_store/blob/master/lib/record_store/provider/dnsimple.rb#L6 should be the correct URL here